safety/Register-Patient-Safety.ps1

```diff
@@ -0,0 +1,272 @@
+        Write-Success "Success."
+        Write-Host ""
+        return $postResponse
+    }catch{
+        $exception = $_.Exception
+        if($exception -ne $null -and $exception.Response.StatusCode.value__ -eq 409)
+        {
+            Write-Success "Entity: "
+            Write-Success $body
+            Write-Success "already exists, skipping creation."
+            Write-Host ""
+        }else{
+            $result = $exception.Response.GetResponseStream()
+            $reader = New-Object System.IO.StreamReader($result)
+            $reader.BaseStream.Position = 0
+            $reader.DiscardBufferedData()
+            $responseBody = $reader.ReadToEnd();
+            Write-Error $responseBody
+            throw $exception
+        }
+    }
+}
+
+function Add-AuthorizationRegistration($authUrl, $clientId, $clientName, $accessToken)
+{
+    $url = "$authUrl/clients"
+    $body = @{
+        id = "$clientId"
+        name = "$clientName"
+    }
+    return Invoke-Post $url $body $accessToken
+}
+
+function Add-Permission($authUrl, $name, $grain, $securableItem, $accessToken)
+{
+    $url = "$authUrl/permissions"
+    $body = @{
+        name = "$name"
+        grain = "$grain"
+        securableItem = "$securableItem"
+    }
+    return Invoke-Post $url $body $accessToken
+}
+
+function Add-Role($authUrl, $name, $grain, $securableItem, $accessToken)
+{
+    $url = "$authUrl/roles"
+    $body = @{
+        name = "$name"
+        grain = "$grain"
+        securableItem = "$securableItem"
+    }
+    return Invoke-Post $url $body $accessToken
+}
+
+function Add-Group($authUrl,$name, $source, $accessToken)
+{
+    $url = "$authUrl/groups"
+    $body = @{
+        id = "$name"
+        groupName = "$name"
+        groupSource = "$source"
+    }
+    return Invoke-Post $url $body $accessToken
+}
+
+function Add-PermissionToRole($authUrl, $roleId, $permission, $accessToken)
+{
+    $url = "$authUrl/roles/$roleId/permissions"
+    $body = @($permission)
+    return Invoke-Post $url $body $accessToken
+}
+
+function Add-RoleToGroup($authUrl, $groupName, $role, $accessToken)
+{
+    $encodedGroupName = [System.Web.HttpUtility]::UrlEncode($groupName)
+    $url = "$authUrl/groups/$encodedGroupName/roles"
+    $body = $role
+    return Invoke-Post $url $body $accessToken
+}
+
+if(!(Test-Path .\Fabric-Install-Utilities.psm1)){
+	Invoke-WebRequest -Uri https://raw.githubusercontent.com/HealthCatalyst/InstallScripts/master/common/Fabric-Install-Utilities.psm1 -OutFile Fabric-Install-Utilities.psm1
+}
+Import-Module -Name .\Fabric-Install-Utilities.psm1 -Force
+
+if(!(Test-IsRunAsAdministrator))
+{
+    Write-Error "You must run this script as an administrator. Halting configuration." -ErrorAction Stop
+}
+
+$installSettings = Get-InstallationSettings "registration"
+$fabricInstallerSecret = $installSettings.fabricInstallerSecret
```

Comment:
Typo `authorizationSerivceURL` (on the left and right side of the assignment)

Reply:
Fixing this.